pandas/tests/reshape/merge/test_merge.py

```diff
@@ -2227,3 +2227,10 @@ def test_categorical_non_unique_monotonic(n_categories):
         index=left_index,
     )
     tm.assert_frame_equal(expected, result)
+
+
+def test_merge_empty_right_index_left_on():
+    left = pd.DataFrame({"a": [1], "b": [2]}).set_index(["a", "b"])
```

Comment:
can you add the issue number here as a comment

Reply:
Done